languagetool-language-modules/nl/src/main/java/org/languagetool/rules/nl/CompoundAcceptor.java

```diff
@@ -558,14 +558,13 @@ public class CompoundAcceptor {
     }
   }
 
-  private final Tagger tagger;
+  private DutchTagger dutchTagger = DutchTagger.INSTANCE;
 
   CompoundAcceptor() {
-    tagger = Languages.getLanguageForShortCode("nl").getTagger();
   }
 
-  public CompoundAcceptor(Tagger tagger) {
-    this.tagger = tagger;
+  public CompoundAcceptor(DutchTagger dutchTagger) {
```

Comment:
This constructor (lines 566-568) seems unnecessary. It is never used, and the tagger is initialized in line 561.